content/docs/communities/moderation/_index.md

```diff
@@ -95,21 +93,280 @@ in the message bar at the bottom:
 /op @yourMjolnirBot:example.com 100
 ```
 
-TODO screenshot of oping mjolnir
+The bot really does need those privileges to be able to actually kick and ban
```

Comment:
It would be nice to explain (the arguably harder) way of having it only Moderator (pl 51 for example) and allowing acl for pl51. That way the bot isn't an admin which always is annoying when you need to remove it later on in case a hs gets unreachable. If it is 51 and has acl perms it still works but admins are also able to remove it when needed. Mostly a quality of life change imho.

Reply:
Obviously this makes it so you can't op people anymore fully with it.

Reply:
(turns out I missed the next few sentences :)

How about documenting it behind a spoiler thingy as an advanced section?

Reply:
Documenting it behind a spoiler is a good idea! I'd like to land a first version quickly though, so I think it makes sense to go without it at first.

I'm also wondering if this should be documented on the Matrix.org "running your own community" documentation or if it should end up in mjolnir's own docs?

Reply:
I feel like they could belong in either or both. A community does necessarily have their own homeserver. And equally a mjolnir mod might have a homeserver they want to moderate with mjolnir.

Reply:
It somewhat only affects mjolnir and not communities in general I think. Like this setup is meant to benefit mjolnir. Maybe it is also helpful for other moderation bots but I feel like it is better in here.